search-service/src/test/kotlin/com/egm/stellio/search/service/EntityQueryServiceTests.kt

```diff
@@ -203,7 +203,7 @@ class EntityQueryServiceTests : WithTimescaleContainer, WithKafkaContainer {
         "integer==213;boolean==true, 1, urn:ngsi-ld:BeeHive:01",
         "(integer>200|integer<100);observedProperty.observedAt<2023-02-25T00:00:00Z, 1, urn:ngsi-ld:BeeHive:01",
         "string~=\"(?i)another.*\", 1, urn:ngsi-ld:BeeHive:02",
-        "string!~=\"(?i)another.*\", 1, urn:ngsi-ld:BeeHive:01",
+        "string!~=\"(?i)another.*\", 2, 'urn:ngsi-ld:BeeHive:01,urn:ngsi-ld:Apiary:05'",
```

Comment:
this raises an interesting point. the `urn:ngsi-ld:Apiary:05` does not have a `string` property, should it be returned? (not to be dealt with as part of this PR)

Reply:
Yes indeed when we do `not like regex` the query returns all entities we don't match with like_regex